From 16e1305131b9cca403f0390f9e4b12a94a8a6b17 Mon Sep 17 00:00:00 2001
From: hundredvisionsguy <cwinikka@gmail.com>
Date: Mon, 1 May 2017 09:27:25 -0700
Subject: [PATCH] fixing Unit tests

---
 TestHeatIndexCalculator.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/TestHeatIndexCalculator.py b/TestHeatIndexCalculator.py
index bf642ef..d2f8e5c 100644
--- a/TestHeatIndexCalculator.py
+++ b/TestHeatIndexCalculator.py
@@ -56,13 +56,13 @@ class KnownValues(unittest.TestCase):
         expected = 132
         self.assertEqual(expected, result)
 
-    def test_calculateHeatIndex_for_80RelativeHumidity_96Temp(self):
-        result = HeatIndexCalculator.calculateHeatIndex(80, 96)
-        expected = 132
+    def test_calculateHeatIndex_for_80RelativeHumidity_94Temp(self):
+        result = HeatIndexCalculator.calculateHeatIndex(80, 94)
+        expected = 129
         self.assertEqual(expected, result)
         
     def test_calculateHeatIndex_for_70RelativeHumidity_98Temp(self):
-        result = HeatIndexCalculator.calculateHeatIndex(80, 96)
+        result = HeatIndexCalculator.calculateHeatIndex(70, 98)
         expected = 134
         self.assertEqual(expected, result)
 
-- 
2.7.1.windows.1

